packages/calypso-products/src/constants/akismet.ts

```diff
@@ -80,3 +80,17 @@ export const AKISMET_PRODUCTS_LIST = <const>[
 	...AKISMET_PLUS_PRODUCTS,
 	...AKISMET_ENTERPRISE_PRODUCTS,
 ];
+
+export const AKISMET_UPGADES_PRODUCTS_MAP = {
```

Comment:
It's a bit late for this, but I just noticed that there is a typo here `UPGADES`

Doesn't keep it from working, but noticed randomly when working on a file that imported this